addon/components/sl-menu.js

```diff
@@ -338,13 +341,28 @@ export default Ember.Component.extend({
     children: null,
 
     /**
-     * Additional class names to be added to the classNames attribute
+     * Array of classes to be added to the element's class attribute
      *
      * @type {String[]}
      */
     extraClassNames: [],
 
     /**
+     * Additional class string to be added to the element's class attribute
+     *
+     * @type {String}
+     */
+    extraClassNamesString: function() {
+        var extraClassNames = this.get( 'extraClassNames' );
+
+        if ( !Ember.isNone( extraClassNames ) ) {
+            return extraClassNames.join( ' ' );
+        } else {
+            return [];
```

Comment:
Oh that's a mistake. It should return an empty string if there are no extraClassNames.